Makefile

```diff
@@ -359,8 +359,8 @@ undeploy-olm: ## Un-deploy the operator with OLM
 DEFAULT_OLM_NAMESPACE ?= openshift-marketplace
 .PHONY: catalog-update
 catalog-update: ## Update catalog source in the default namespace for catalogsource
-	kubectl delete catalogsource backstage-operator -n $(DEFAULT_OLM_NAMESPACE)
-	sed "s/{{CATALOG_IMG}}/$(subst /,\/,$(CATALOG_IMG))/g" config/samples/catalog-source-template.yaml | kubectl apply -n $(DEFAULT_OLM_NAMESPACE) -f -
+	-kubectl delete catalogsource backstage-operator -n $(DEFAULT_OLM_NAMESPACE)
+	-sed "s/{{CATALOG_IMG}}/$(subst /,\/,$(CATALOG_IMG))/g" config/samples/catalog-source-template.yaml | kubectl apply -n $(DEFAULT_OLM_NAMESPACE) -f -
```

Comment:
why is it OK for these to fail silently?

Reply:
because it the delete fails the catalogsource did not exist in the namespace yet. Without this change we won't be able to use "catalog-update" for a fresh cluster.

Reply:
+1 I had the same issue earlier today when running `make catalog-update` on a new cluster.
But maybe only the `kubectl delete catalogsource ...` could fail silently, but the `sed ... | kubectl apply ...` should not, no?

Reply:
Right, kubectl apply should not fail.